openedx/core/djangoapps/discussions/models.py

```diff
@@ -145,6 +180,9 @@ class Features(Enum):
             '',
             '',
         )._asdict(),
+        "messages": ProviderMessages(
```

Comment:
I can still see `ProviderExternalLinks` props have not been updated to use kwargs.

Reply:
Done!